### PR TITLE
readme updates and io test update

### LINKED ISSUE #14: Fix Unit Test
io unit test in test.py needs a name change to 'test_io' and needs to be fixed

### LINKED ISSUE #15: QA Check
Final review of the repo before submission

### PR DESCRIPTION
**Pull request by edouarde1**
closes #14 #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,19 +24,20 @@ The functions created are as follows:
...
 
+Uses txt_read() and scrambles the words in the output
 
 ### txt_read()
-Arguments : a String containing the name of the txt file you wish to read. 
+
+Arguments : a String containing the name of the txt file you wish to read.
 Outputs: A string containing the wors of the txt file.
 
 Reads a .txt and returns following data as a string.
```
Comment (LoganParker) [inline README.md:43]: Looks good, these changes will be valuable for clarity when looking back on the project :)

File: test.py [hunk 2]
```diff
@@ -3,9 +3,9 @@
 from wordsoup import wordsoup, txt_read
 
 
-class MainTestCase(unittest.TestCase):
-    def io_test(self):
-        self.assertEqual('Our team had a debate for best names for looping variables ? i won',
+class GroupExerciseTest(unittest.TestCase):
+    def test_io(self):
+        self.assertEqual('Our team had a debate for best names for looping variables ? i won\n',
```
Comment (LoganParker) [inline test.py:8]: Good catch Mr. QA, glad your test case works now 👍

**Review by LoganParker — APPROVED** [review]
Everything looks great, those readme changes are very valuable

File: README.md [hunk 3]
```diff
@@ -24,19 +24,20 @@ The functions created are as follows:
 4. Sorter function to sort words in a body  of text alphabetically.
 5. Randomizer that can that scramble words randomly in a body of text.
 
-## Features
+## Testing Functions
 
-This program provides a series of functions that are helpful for file preprocessing and file analysis.
+The following functions are used for testing purposes.
```
Comment (vnajack) [inline README.md:29]: nice and concise

File: test.py [hunk 4]
```diff
@@ -3,9 +3,9 @@
 from wordsoup import wordsoup, txt_read
 
 
-class MainTestCase(unittest.TestCase):
-    def io_test(self):
-        self.assertEqual('Our team had a debate for best names for looping variables ? i won',
+class GroupExerciseTest(unittest.TestCase):
+    def test_io(self):
```
Comment (vnajack) [inline test.py:7]: glad you got it working alright. Good to know that the python unit test cases require function names that start with test_

**Review by vnajack — APPROVED** [review]
Thank you for doing a QA check!

**Pull request merged**